spec/features/scheduling/schedule_spec.rb

```diff
@@ -1,82 +1,68 @@
 # frozen_string_literal: true
 
-RSpec.feature "Schedule an edition" do
+RSpec.feature "Schedule a publishing" do
```

Comment:
Minor, but neither of these make much sense to me 😆 . How about 'Schedule to publish'?

Reply:
I don't really know how this makes much difference but I've gone with your suggestion